yet-another-docker-plugin/src/main/java/com/github/kostyasha/yad/launcher/DockerComputerJNLPLauncher.java

```diff
@@ -287,21 +316,36 @@ public ComputerLauncher getLauncher() {
     }
 
     @Override
-    public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, CreateContainerCmd createContainerCmd)
+    public void appendContainerConfig(DockerSlaveTemplate dockerSlaveTemplate, CreateContainerCmd createContainerCmd, String osType)
```

Comment:
i feel that will be not the last variable... 
Could you make some `AdditionalInfo` class and move `osType` to it?